pkg/workspace/inference/preset-inferences.go

```diff
@@ -25,7 +25,12 @@ import (
 
 const (
 	ProbePath = "/health"
-	Port5000  = 5000
+
+	// Port5000 is the default port for vLLM inference server.
+	Port5000 = 5000
```

Comment:
give it a name. PortInferenceServer.
transformers and vllm share the same port.